packages/components/src/components/filter-dropdown/filter-dropdown.tsx

```diff
@@ -1,26 +1,38 @@
 import * as React from 'react';
-import PropTypes from 'prop-types';
 import { isMobile } from '@deriv/shared';
-import Dropdown from '../dropdown/dropdown.jsx';
+import Dropdown from '../dropdown/index.js';
```

Comment:
```suggestion
import { Dropdown } from '../dropdown/dropdown.jsx';
```

Reply:
Yeah i tried this one too.Still it was showing same error.